scripts/zones/Valley_of_Sorrows/mobs/Adamantoise.lua

```diff
@@ -3,13 +3,23 @@
 --  HNM: Adamantoise
 -----------------------------------
 local ID = zones[xi.zone.VALLEY_OF_SORROWS]
-mixins = { require('scripts/mixins/rage') }
+mixins =
+{
+    require('scripts/mixins/rage'),
+    require('scripts/mixins/draw_in'),
+}
 -----------------------------------
 ---@type TMobEntity
 local entity = {}
 
 entity.onMobSpawn = function(mob)
     mob:setLocalVar('[rage]timer', 1800) -- 30 minutes
+    mob:addImmunity(xi.immunity.LIGHT_SLEEP)
+    mob:addImmunity(xi.immunity.DARK_SLEEP)
+    mob:setMod(xi.mod.DMGMAGIC, -3500)
+    mob:setMobMod(xi.mobMod.WEAPON_BONUS, 36) -- 108 total weapon damage
+    mob:setMod(xi.mod.DEF, 4112)
```

Comment:
4112, id this number correct?

Reply:
Value is taken from Jimmayus capture work. The defense value for Adamantoise is listed as "5000 (yes)" on the HNM stats page there